qcengine/programs/psi4.py

```diff
@@ -7,7 +7,8 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, ClassVar, Dict
 
-from qcelemental.models.v2 import AtomicResult, BasisSet
+from qcelemental.models.v1 import AtomicResult as AtomicResult
```

Comment:
It looks like `as AtomicResult` can be removed.

Reply:
Thanks, that's the second time I've messed that line up. First time was something like `... import AtomicInput as AtomicResult`, which was confusing. I've fixed this and added a comment for your previous comment in the next PR.